staging/src/k8s.io/kubelet/pkg/types/labels.go

```diff
@@ -24,6 +24,13 @@ const (
 	KubernetesContainerNameLabel = "io.kubernetes.container.name"
 )
 
+// Label value constants
```

Comment:
Do you think we should add these labels to https://kubernetes.io/docs/reference/labels-annotations-taints/?

Reply:
The linked labels are for end users while the ones from the staging are more for API consumers, so I'm leaning towards a "no" here.

Reply:
The linked labels are labels that kubernetes uses/adds.  Its mostly to document known labels that Kubernetes uses.  For the job controller, we would also document any labels that Kubernetes adds to pods.

We also have labels like diskpressure and other items that kube adds to nodes.  Its not a requirement for this PR I think but it may be nice to document these labels on pods because I didn't see any of these in that list.

Reply:
These labels are not user facing, means they're not visible via `kubectl`. But you can inspect them via `crictl`, like:

```
> sudo crictl inspect $CTR | jq .status.labels
{
  "io.kubernetes.container.name": "coredns",
  "io.kubernetes.pod.name": "coredns-69cbfb9798-6lj8k",
  "io.kubernetes.pod.namespace": "kube-system",
  "io.kubernetes.pod.uid": "c78368f2-3d20-404f-b82e-718df30da577"
}
```